README.md

```diff
@@ -1,92 +1,111 @@
-# SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure
+# SAS Viya Infrastructure as Code (IaC) for Microsoft Azure
 
 ## Overview
 
-This project contains Terraform scripts to provision Microsoft Azure Cloud infrastructure resources required to deploy SAS Viya 4 products. Here is a list of resources this project can create -
+This project contains Terraform scripts to provision the Microsoft Azure Cloud infrastructure resources that are required to deploy SAS Viya product offerings. Here is a list of resources that this project can create:
```

Comment:
Are the references to SAS Viya going to be reverted to SAS Viya 4 in the doc? If not and the company is moving to the more generic SAS Viya I am fine with that.